experimental/ni_coq/README.md

```diff
@@ -18,10 +18,13 @@ is in vfiles/RuntimeModel.v. Both are evolving.
 
 ## Instructions
 
-The following are dependencies of this project: coq (8.11.0) coq-mathcomp-finmap
-(1.5.0)
+The following are dependencies of this project:
 
```

Comment:
It will be better if we mention adding a coq package repo is desired to install Coq packages with opam: "opam repo add coq-released https://coq.inria.fr/opam/released"

Reply:
Updated in latest commit.